wrappers/python/tests/layer/test_act.py

```diff
@@ -11,13 +11,15 @@
 #
 # @version 1.0.0
 
-# All necesary imports
-import nntile
 import numpy as np
-import torch.nn.functional as F
 import torch
+import torch.nn.functional as F
+
+# All necesary imports
+import nntile
+
 # Set up StarPU configuration and init it
-config = nntile.starpu.Config(1, 0, 0)
+config = nntile.starpu.Config(1, 1, 0)
```

Comment:
idea of parameters (1, 0, 0) is to test only on CPU, as CUDA testing shall be done in tests/kernel. This test shall only test wrappers.